django_mongodb/utils.py

```diff
@@ -41,12 +40,11 @@ def profile_call(self, func, args=(), kwargs=None):
         return duration, retval
 
     def log(self, op, duration, args, kwargs=None):
+        # If kwargs are used by any operations in the future, they must be
+        # added to this logging.
```

Comment:
What's the likelihood that any operations will be added here in the future?

Reply:
I'm not sure. You can look at the kwargs for [`update_many()`](https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.update_many), for example, and see if you think developers will need some way to specify any of them for QuerySet operations.